### DISCUSSION TITLE
Move cards in project to top/bottom

### DISCUSSION THREAD

**Discussion by EJucys** [community/community#4757] [Projects and Issues]
Hi there, 

As a Product Owner who's actually managing stuff in Github and not other product management tools, I really miss easier the card dragging option. Especially when there are >10 cards and you want to move the lowest (newest) one to the top as a priority card. 
I'd love it if we could have the card moving to top/bottom as I feel this would improve `/projects` immensely. 

<img width="353" alt="Screenshot at Jul 22 18-37-34" src="https://user-images.githubusercontent.com/23452816/126667785-fe486c7b-c8de-4a2e-b525-c933331c7709.png">

**Comment by amacneil** [comment]
Agreed, this small feature would improve things a lot.

**Comment by kylos101** [comment]
Yes, it would be helpful to have move to top/bottom shortcuts. [These shortcuts do not appear to work in Projects Beta.](https://docs.github.com/en/get-started/using-github/keyboard-shortcuts#moving-a-card)

As a workaround, because order denotes priority, I'm dragging cards on my portrait (tall) monitor. Not ideal, but waaay better than the landscape view.

**Comment by EzimetYusup** [comment]
Would it be possible to sort the cards based on creation date? 
I wish we can do it via GitHub API at least, but looks like there is no api to support sorting yet.

**Comment by nbarbettini** [comment]
I am surprised this doesn't have any traction yet. For a large project, being able to move cards to the top/bottom is a ~~must-have~~ strong nice-to-have!

**Comment by jfritz** [comment]
Bumping this request...

If you have a backlog of hundreds of items, how are you expected to order your backlog by priority -- a foundational pillar of Scrum -- click and drag each one? We need ordering shortcuts in the UI and maybe even API support for ordering items.

Basic product refinement features are missing, GH Projects Beta seems geared toward prioritizing one time issues or bugs...

In the Table view:

First and foremost, I'd like to see "Move to Top" and "Move to Bottom". 
I'd like to see "Split Issue" in the right click context menu.
I'd like to also see "Move to Sprint X" in the right click context menu.

**Comment by nbarbettini** [comment]
@jfritz I noticed recently that GH Projects beta added Move to Top/Bottom and Move to Column options:
![image](https://github.com/community/community/assets/7525482/8a3226ca-2901-4e84-8988-e2795dccb350)

**Reply by jfritz** [reply]
This is in board/card view only, which is a start, but is the wrong place to implement this first in my opinion.

MVF should be implementing this in table view, as it is a power user view for product owners to easily order things in priority order.

Cards on the board are too bulky to prioritize en masse.

**Reply by nbarbettini** [reply]
Ah, I missed that you were talking about Table view. Makes sense!

**Comment by lifehackett** [comment]
+1

**Comment by svix-onelson** [comment]
We're currently switching views from Table Layout over to Board Layout, _just to be able to move to top_, then switch back again.

Would be very nice to not need to do this 🤣
